---
content_type: page
description: This section contains lecture summaries for every week.
hide_download: true
hide_download_original: null
learning_resource_types:
- Lecture Notes
ocw_type: CourseSection
title: Lecture Summaries
uid: bf331a25-b5c6-b71c-88e9-399f06353e46
---

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
week #
{{< thclose >}}
{{< thopen >}}
topics
{{< thclose >}}
{{< thopen >}}
lecture summaries
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
1
{{< tdclose >}}
{{< tdopen >}}
Overview
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "L1summary" >}}{{< /anchor >}}The semester will begin with all of us introducing ourselves. The instructors will then (i) present a general overview of the course format, the course website, and assignments, (ii) define objectives of the course, and (iii) explain what is expected from the students. We will also discuss how to do literature searches. Instructors will then kickstart the course by presenting the core diagram of the landmark “Operon paper” by Jacob and Monod from 1961. This paper predicted the existence of messenger RNA (mRNA) and correctly inferred major modes of gene regulation. Instructors will explain the original open questions at that moment and prepare the students to read two classic research papers that helped to establish that information flows from DNA to protein through an unstable intermediate, messenger RNA. We will take the opportunity to modify Jacob & Monods’ original diagram to tease at RNA functions that were discovered later. 
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
2
{{< tdclose >}}
{{< tdopen >}}
From the discovery of mRNA to the “central dogma”
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "L2summary" >}}{{< /anchor >}}This week we will discuss what molecular biologist mean by the “central dogma” and the predominant hypothesis at the time which proposed that each gene expressed the specific “microsomal” RNA that would then determine the protein. We discuss how Astrachan & Volkin used nucleotide composition to show that newly transcribed RNA after infection originates from the phage genome. This paper offers a great example of how the combination of a perturbation (infection) and a measurement (composition of labeled RNA) can lead to mechanistic insights. We then discuss how Brenner, Jacob, and Meselson were able to synthesize these and other observations into a coherent model for gene expression by systematic exclusion of the alternative hypotheses that protein is made from DNA directly or that the “microsome” (ribosome) is gene-specific. We will see how these classic experiments relied on measurements of bulk RNA and cleverly exploited labeling time to study RNA components with different dynamics.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
3
{{< tdclose >}}
{{< tdopen >}}
From the discovery of splicing to studying entire “transcriptomes”
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "L3summary" >}}{{< /anchor >}}We will start by explaining R-loop mapping: the identification of the genomic origin of cellular transcripts by hybridization of the RNA to the genomic, double-stranded DNA. From Berget et al. we will discuss how mRNA has been studied using electron microscopy. We will learn about the likely evolution of introns from self-splicing introns and their effect on the evolution of host genomes. From Nagalakshmi et al. we will highlight the dramatic paradigmatic shift brought about by nucleic acid sequencing, which in principle allows the study of any desired subset of RNAs, down to specific genes, and (in its highthroughput version) generates catalogs of entire “transcriptomes” from RNA-preparations. We discuss how RNA splicing adds a new layer for potentially regulating gene expression and how RNA-sequencing allows the study of species on a large scale. 
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
4
{{< tdclose >}}
{{< tdopen >}}
What is a gene really? From baby RNAs to mature mRNAs
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "L4summary" >}}{{< /anchor >}}How much RNA do you think is made in a human cell? There are many processing steps before an mRNA molecule can be used to make proteins. One of these steps is the removal of introns from the mRNA and the joining of the adjacent exons—splicing. Splicing is controlled by the action of small nuclear RNAs (snRNAs) that form a large, multi-subunit complex called the spliceosome. From Merkin et al. we will discuss how the removal of specific introns can vary among tissues, species, and conditions. Moreover, alternative splicing patterns differ more among species than do gene expression patterns. From de la Mata et al. we will discuss the mechanisms of splicing. Splicing is mostly co-transcriptional, leading to the possibility of functional and temporal connections between transcription and splicing. Indeed, transcription dynamics, e.g., the elongation rate of RNA Polymerase II, can affect alternative splicing choices.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
5
{{< tdclose >}}
{{< tdopen >}}
How mRNAs are synthesized
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "L5summary" >}}{{< /anchor >}}The promoter of a gene is a region of DNA that initiates transcription. From Schor et al. we will consider how promoters come in a two shapes (sharp or broad), vary across populations, and directly affect gene expression. Things started to get more complicated when it was realized that divergent transcription, in which reverse-oriented transcripts occur upstream of eukaryotic promoters, is a general property of active promoters. It took many years to understand what is going on. From Duttke et al., we will discuss the current idea that divergent transcription is not a property of the transcription process but rather the consequence of the presence of both forward- and reverse-directed core promoters.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
6
{{< tdclose >}}
{{< tdopen >}}
How you start is how you proceed: promoter control of alternative splicing and processing
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "L6summary" >}}{{< /anchor >}}This week we will learn that RNA processing does not depend exclusively on signals present in the pre-RNA molecule. We will introduce the super exciting topic of possible links between the promoter and alternative events further downstream in the gene body. From Cramer et al. we will discuss how changes in promoter structure strongly affect splice-site selection. We will see evidence for coupling between alternative splicing and promoter-specific transcription, which agrees with the most recent idea of coordination between splicing and transcription. Furthermore, from Chen et al. we will go back to promoter structure and discuss how the distance between promoters correlates with the expression, stability, and length of their associated upstream antisense RNAs, which are formed by divergent transcription at the majority of RNA polymerase II promoters.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
7
{{< tdclose >}}
{{< tdopen >}}
miRNAs: small RNAs making a big splash
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "L7summary" >}}{{< /anchor >}}The specificity of base-pairing allows DNA and RNA to copy and carry genetic information. But it also allows for specific intermolecular interactions (for example tRNAs decoding an mRNA, or within the spliceosome). Often, intermolecular interactions occur in the context of proteins that can constrain the nucleic acids and optimize their properties for binding. Argonaute proteins are an important class of such proteins, found across eukaryotes and capable of loading a short RNA “guide” such as a microRNA (miRNA). The loaded Argonaute can then bind and repress mRNAs with sequence complementarity to the 5’ region of the miRNA (the “seed” region). Such miRNA binding sites are typically found in 3’ untranslated regions (3’UTR), downstream of the stop codon.

Lee et al. describe their discovery of a genetic interaction between a C. elegans protein coding gene and a non-coding RNA (a microRNA) called lin-4. Their experiments demonstrate that lin-4 represses expression of lin-14 protein. They characterize the small RNA product of lin-4 and, by analyzing its sequence, are able to infer that it binds lin-14 mRNA via an antisense RNA-RNA interaction: basepairing between the miRNA and complementary sequence in the mRNA (a miRNA binding site). This finding briefly brings us back to the Operon paper, because regulation at the RNA level by miRNAs fits into Scenario II hypothesized by Jacob and Monod.

The second paper is very recent and addresses details of the miRNA-mRNA interaction, which we now know happens in the context of Argonaute protein. By designing reporter assays with specifically designed mutations affecting the complementarity between miRNA and target, Flamand et al. investigate the role of binding sites with imperfect seed matches and cooperativity between target sites, a field of ongoing research.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
8
{{< tdclose >}}
{{< tdopen >}}
Field trip to [Arrakis Therapeutics](http://arrakistx.com/)
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "L8summary" >}}{{< /anchor >}}To glimpse into state-of-the art RNA technologies for medical applications, we will visit the biotechnology company [Arrakis Therapeutics](http://arrakistx.com/). Students will learn how the company is using bioinformatics and chemical biology tools combined with medicinal chemistry to discover RNA-targeting small molecules (rSMs) and develop drugs for treating neurological disorders, cancer, and rare genetic diseases.  
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
9
{{< tdclose >}}
{{< tdopen >}}
siRNAs: How silencing almost any gene became easy and cheap
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "L9summary" >}}{{< /anchor >}}Some of the Argonaute proteins we have learned about are capable of an additional function. If complementarity between the guide RNA and target is very extensive, the protein can induce cleavage of the mRNA. This is called slicing activity and is not normally induced by miRNAs in animals but can be a potent mechanism to degrade unwanted messenges. Should the targeted RNA derive from a pathogen, it would be even better if the guide RNAs could be copied and amplified. Indeed, some animals, such as C. elegans, have RNA-dependent RNA polymerases. In such animals, exogenous introduction of artificial RNA matching an endogenous gene leads to the strong suppression of this gene in almost every cell. This phenomenon is called RNA interference (RNAi for short). RNAi can also be exploited in the lab by introducing artificial small interfering RNAs (siRNAs) into cells to silence the expression of almost any gene. This approach has become so commonplace that robots can perform RNAi against thousands of candidate genes, monitoring the activity of a reporter, to find those genes that change the activity of a specific mechanism under investigation, an siRNA screen.

From the first paper, we learn how Fire et al. systematically tested the requirements for RNAi in C.elegans, where it is potent and systemic (spreading through the animal). By carefully varying the experimental conditions and exploiting genetics, the authors were able to correctly infer the existence of catalytic activity and amplification in the RNAi-pathway of the animal. Many years later, Jenal et al. used thousands of artificial small-interfering RNAs (siRNAs) to identify human genes that affect alternative poly-adenylation, a mechanism that creates mRNA isoforms with different 3’UTRs from the same gene. This approach demonstrates the utility of siRNA-based screens to identify the molecular components of a cellular pathway. We will discuss the potential consequences of alternative 3’ ends for post-transcriptional gene regulation based on what we have learned about miRNAs.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
10
{{< tdclose >}}
{{< tdopen >}}
It’s after the stop, yet somehow on top. How the 3’UTR regulates the function of encoded proteins
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "L10summary" >}}{{< /anchor >}}In principle, regulatory cis-elements in 3’UTRs could functionally interact in complicated ways: for example, we have learned that nearby binding sites for miRNAs can cooperate. Binding sites for miRNAs and RNA-binding proteins could also overlap or be parts of RNA structures. All this can constrain the function of regulatory sequence elements and make the activity of one element depend on the activity of others. Fluorescent proteins are a useful tool to measure regulatory activity because they allow the comparison of the activities of different 3’UTR sequences on total protein output or on the distribution of the protein within a cell.

Kristjánsdóttir et al. “disassembled” a long 3’UTR into short fragments and tested each for its effects on protein production using a fluorescent reporter assay. This approach allowed the identification of a large number of functional elements in the Hmga2 3’UTR and revealed that these elements act largely independently from one another. Next we learn about the discovery that 3’UTR choice can alter not the amount but also the function of the protein. By combining carefully designed reporter constructs with microscopy, flow cytometry, and RNA interference, Berkovits and Mayr were able to illuminate a chain of molecular events that switch the localization and function of CD47 protein, depending on whether it is translated from an mRNA isoform harboring either the short or long 3’UTR (with identical coding sequences).


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
11
{{< tdclose >}}
{{< tdopen >}}
Raise the shields (and scissors)! piRNA and CRISPR are defensive RNA-weapon systems
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "L11summary" >}}{{< /anchor >}}Viruses replicate by inserting their nucleic acids into a cell, which allows them to hijack the host’s machinery for protein synthesis and copying of their genomes. Transposons derive from such viruses that have lost the ability to leave an infected cell. Transposons can insert into the genome of the host’s germ cells, become heritable, and still spread within the genome of the host. Fascinating defense mechanisms against invading foreign sequences have evolved across life.

First we will look at piwi-interacting RNAs (piRNAs). Brennecke et al. sequenced small-RNAs from Drosophila to study piRNAs, distant relatives of miRNAs and siRNAs. piRNAs are highly expressed in germ cells. The authors found that piRNAs originate from broken copies of transposons, inserted at specific sites within the genome. Using fly genetics, they demonstrated that a few of these sites are “master regulators” of transposon activity and elucidated the mechanism by which piRNAs form an adaptive immune system that can keep transposons in check.

We will then transition to a conceptually similar—but completely independent—system from prokaryotes, which are under constant attack from phages (viruses for bacteria). Jinek et al. demonstrated that the bacterial CRISPR-Cas9 system is an RNA-programmed DNase, capable of shredding the DNA of incoming phages. By identifying the functional elements in the two RNA components used in the natural system, the authors were able to engineer a single guide RNA (gRNA) sufficient to program Cas9 to induce DNA-double strand breaks at sequences of choice. We will mention the long history of prior research that led to this amazing new tool for genome editing and point to some recent advances using CRISPR-based systems for lineage tracing or recording information about a cell’s past.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
12
{{< tdclose >}}
{{< tdopen >}}
Easy to miss, hard to degrade: circular RNA and lncRNAs
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "L12summary" >}}{{< /anchor >}}Although many crucial catalytic and structural molecules in the cell are proteins, any RNA biologist will tell you that RNA molecules have been fulfilling these roles for billions of years. RNA molecules might be ideal scaffolds for regulatory complexes, particularly those that involve the regulation of gene expression, as they can contain both binding sites for many proteins and the ability to hybridize with other nucleic acids. Single-stranded circular RNA molecules without any encapsulation were found to be infectious agents in plants, capable of replication and spreading to other plants.

We will briefly discuss the history of how (endogenous) circular RNAs (circRNAs) were discovered in other organisms, including mammals. While many circRNAs could be inconsequential by-products of splicing, a few are now known to have a biological function. Piwecka et al., made mice without Cdr1as, a circRNA with high expression in the brain. Interestingly, they find very specific alterations of miRNA function and behavior. In addition, we have seen that RNA molecules have the ability to silence expression at the level of RNA. From Chaumeit et al. we will discuss how the long non-coding RNA (lncRNA) Xist takes this repression to the level of DNA by repressing gene expression across almost an entire chromosome. By acting as a scaffold for transcriptional repressors and chromatin-modifying enzymes, Xist is able to efficiently shut off transcription of many genes at once.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
13
{{< tdclose >}}
{{< tdopen >}}
Integration of classes, writing assignment, and course discussion
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "L13summary" >}}{{< /anchor >}}In addition to student oral presentations this week, we will end class with a discussion of the course as a way for students to provide feedback. Course evaluations will be completed.
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}